### PR TITLE
{Servicebus} `az servicebus topic subscription rule`: Migrate servicebus rule cmdlets to AAZ

### PR DESCRIPTION
**Pull request by schaudhari6254888**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Servicebus

**Comment by schaudhari6254888** [comment]
This PR is for Migrating ServiceBus rule cmdlets to aaz.
Can someone please look into is & review this.
Thank you!

File: .github/azure-client-tools-bot/config.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+auto_choose_milestone:
+  enable: True
+  issue_expected_days: 7
+  pr_expected_days: 7
```
Comment (evelyn-ys) [inline .github/azure-client-tools-bot/config.yml:4]: Is this related with your migration? Did this come from some other's commit?

File: src/azure-cli-core/azure/cli/core/aaz/_command.py [hunk 2]
```diff
@@ -447,7 +451,13 @@ def _load_aaz_pkg(loader, pkg, parent_command_table, command_group_table, arg_st
         try:
             sub_pkg = importlib.import_module(f'.{sub_path}', pkg.__name__)
         except ModuleNotFoundError:
+            logger.debug('Failed to load package folder in aaz: %s.', os.path.join(pkg_path, sub_path))
             continue
+
+        if not sub_pkg.__file__:
+            logger.debug('Ignore invalid package folder in aaz: %s.', os.path.join(pkg_path, sub_path))
+            continue
```
Comment (evelyn-ys) [inline src/azure-cli-core/azure/cli/core/aaz/_command.py:459]: cc @kairu-ms for reviewing

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 3]
```diff
@@ -495,6 +496,20 @@ def aks_create(
...
+        existing_mc = client.get(resource_group_name, name)
+    # pylint: disable=broad-except
+    except Exception as ex:
+        logger.debug("failed to get cluster, error: %s", ex)
+    if existing_mc:
+        raise ClientRequestError(
+            f"The cluster '{name}' under resource group '{resource_group_name}' already exists. "
+            "Please use command 'az aks update' to update the existing cluster, "
+            "or select a different cluster name to create a new cluster."
+        )
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py:511]: Why modify acs module in service bus migration PR?

**Review by evelyn-ys — COMMENTED** [review]
Can you rebase your branch, I can't tell which file changes are related with your migration and which are not

**Comment by schaudhari6254888** [comment]
> Can you rebase your branch, I can't tell which file changes are related with your migration and which are not

Hey @evelyn-ys, sorry for the inconvenience. I have rebase my branch Now!

**Review by evelyn-ys — APPROVED** [review]
